tubular/tests/test_frontend_build.py

```diff
@@ -62,5 +64,9 @@ def test_frontend_build_config_handling(
             "BOOL_FALSE_WITH_QUOTES='False'",
             "NONE='None'",
             "NONE_WITH_QUOTES='None'",
+            "JS_CONFIG_FILEPATH='dummy/file/path/env.config.js'",
         ]
         assert mock_create_version.call_count == 1
+        assert mock_shutil_copyfile.call_count == 1
+        # Verify that source is correct and destination is rightly formatted
+        mock_shutil_copyfile.assert_called_with('dummy/file/path/env.config.js', 'coolfrontend/webpack.prod.config.js')
```

Comment:
Adding onto Max's question above about `LOCAL_CONFIG_FILENAME`, the copied file should be named `env.config.js` within the destination path (i.e., root of the MFE). The copied file should not be the Webpack configuration.